downloads/swan-lake-release-notes/swan-lake-2201.1.2/RELEASE_NOTE.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.1.2 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201.1.2/
+active: 2201.1.2
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.1.2
+    - /downloads/swan-lake-release-notes/2201.1.2/
+    - /downloads/swan-lake-release-notes/2201.1.2-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.1.2-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.1.2
+
+<em>Swan Lake 2201.1.2 is the second patch release of Ballerina 2201.1.0 (Swan Lake Update 1) and it includes a new set of bug fixes to the compiler, runtime, standard library, and developer tooling.</em> 
+
+## Update Ballerina
+
+**If you are already using Ballerina 2201.0.0 (Swan Lake)**, run either of the commands below to directly update to 2201.1.2 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/).
+
+`bal dist update` (or `bal dist pull 2201.1.2`)
+
+**If you are using a version below 2201.0.0 (Swan Lake)**, run the commands below to update to 2201.1.2.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+
+2. Run `bal dist update` ( or `bal dist pull 2201.1.2`) to update your Ballerina version to 2201.1.2.
+
+However, if you are using a version below 2201.0.0 (Swan Lake) and if you already ran `bal dist update` (or `bal dist pull 2201.1.2`) before `bal update`, see [Troubleshooting](/downloads/swan-lake-release-notes/2201-0-0-swan-lake/#troubleshooting) to recover your installation.
+
+## Install Ballerina
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.1.2%22+is%3Aclosed+label%3ATeam%2FCompilerFE).
```

Comment:
```suggestion
To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+label%3ATeam%2FCompilerFE+milestone%3A2201.1.2).
```

Reply:
@keizer619 hope this is correct?

Reply:
Updated.

Reply:
@dulajdilshan @NipunaMadhushan - Please confirm if this link is correct.

Reply:
@praneesha Yes this is correct.

Reply:
Suggested changes are correct